ansible/roles/kubernetes/files/make-ca-cert.sh

```diff
@@ -102,7 +102,7 @@ cd easy-rsa-master/easyrsa3
 
 # Sadly, openssl is very verbose to std*err* with no option to turn it off.
 if ! (./easyrsa --batch init-pki
-      ./easyrsa --batch "--req-cn=${cert_ip}@$(date +%s)" build-ca nopass
+      ./easyrsa --batch "--req-cn=$(echo $(echo ${cert_ip} | cut -b 1-$(expr 64 - $(echo @$(date +%s) | wc -c)))@$(date +%s))" build-ca nopass
```

Comment:
Would you be willing to move this to anther line with a comment about what it is doing and why?